homeassistant/components/incomfort/__init__.py

```diff
@@ -10,6 +10,7 @@
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import ConfigEntryAuthFailed
 
+from .const import DOMAIN  # noqa: F401  # ignore unused
```

Comment:
Why did you add it back in?
If it's for tests, it would be better to point the tests to "const"

Reply:
I'll change that